rules/toolkit-cpp-windows.ham

```diff
@@ -0,0 +1,122 @@
+if $(BUILD) = da {
```

Comment:
See other comments, I'd start from `pierre-linux-clang` (PR #10) it should be very similar to this.